### DISCUSSION TITLE
SDM630-V2 Data

### DISCUSSION THREAD

**Discussion by rolkef** [johanmeijer/grott#293] [Q&A] [answered]
Hi,
I'm using a MIC 3300 TL-X (Inverter) and an ShineWifi-X Datalogger, also an Smartmeter (SDM630-V2). I get the data sent to the Growatt APP (including the Smartmeter values) but i dont get it in my mqtt message.

Its automatically using the T060120 Layout which not reads data like Exporting and Import Energy from the provider. Is there some work-around to get the smartmeter data? :)

thanks!

**Comment by johanmeijer** [comment]
I am sure (not using a smartmeter myself). Can you sent the original Data from the t060120 record from the log  (with the /x in it).
So we can look what is in it?

**Reply by rolkef** [reply]
Hi thanks for your fast reply. Following the log. As you can see at first it detected it with another layout, at the end it used the t060120 record. Another question, is there a way to set the record fixed?
[grott_log.txt](https://github.com/johanmeijer/grott/files/10776325/grott_log.txt)

**Comment by johanmeijer** [comment, marked as answer]
Looks good to me. What is your problem? 

The first record ( T060104X) is the solar panel data. The second the sdm630. 

You can get some more information bij specifing includeall = True in grott.ini but I am not sure that is where you looking for.

**Reply by rolkef** [reply]
Ah thanks didnt know that!
